Kickstarter-iOS/Views/Controllers/ManagePledgeViewControllerTests.swift

```diff
@@ -19,12 +19,17 @@ final class ManagePledgeViewControllerTests: TestCase {
     super.tearDown()
   }
 
-  func testView() {
+  func testView_CurrentUser_IsBacker() {
```

Comment:
Is this just future-proofing? As I understand it from our discussions the user will always be the backer? Which is why I removed the creator strings in #934 🤔

Reply:
Yeah sort of - although there's currently no way to get to this state, the VM does support it so I created a screenshot so we can verify that it visually matches what we expect.